contracts/votes/src/contract.rs

```diff
@@ -367,4 +383,10 @@ impl SorobanOnly for TokenVotes {
     fn admin(e: Env) -> Address {
         storage::get_admin(&e)
     }
+
+    #[cfg(not(feature = "sep-0041"))]
+    fn balance(e: Env, id: Address) -> i128 {
+        storage::extend_instance(&e);
+        storage::get_balance(&e, &id)
+    }
```

Comment:
Can we make a `TokenData` trait that defines `balance`, `name`, `symbol`, and `decimals`?

We can implement it for all contracts where `#[cfg(not(feature = "sep-0041"))]`, to remove the duplicated definitions of those functions in the `Admin` and `Bonding` trait